vhdbuilder/packer/test/windows-vhd-content-test.ps1

```diff
@@ -179,6 +179,56 @@ function Test-FilesToCacheOnVHD
 
 }
 
+function Test-ValidateAllSignature {
+    foreach ($dir in $map.Keys) {
+        # Skip the validation of k8s pacakges since win-bridge.exe is not signed yet
+        if ($dir.StartsWith("c:\akse-cache\win-k8s\")) {
+            continue
+        }
+        Test-ValidateSinglePackageSignature $dir
+    }
+}
+
+function Test-ValidateSinglePackageSignature {
+    param (
+        $dir
+    )
+
+    foreach ($URL in $map[$dir]) {
+        $fileName = [IO.Path]::GetFileName($URL)
+        $dest = [IO.Path]::Combine($dir, $fileName)
+
+        if(![System.IO.File]::Exists($dest)) {
+            Write-ErrorWithTimestamp "File $dest does not exist"
+            $invalidFiles = $invalidFiles + $dest
+            continue
+        }
+
+        $installDir="c:\SignatureCheck"
+        New-Item -ItemType Directory $installDir -Force | Out-Null
+        if ($fileName.endswith(".zip")) {
+            # Skip v0.0.31 and v0.0.32 of windows cse script package since some binaries in them are not signed
+            if ($fileName.endswith("v0.0.31.zip") -or $fileName.endswith("v0.0.32.zip")) {
+                continue
+            }
+            Expand-Archive -path $dest -DestinationPath $installDir -Force
+        } elseif ($fileName.endswith(".tar.gz")) {
+            tar -xzf $dest -C $installDir
+        } else {
+            # Skip the validation of github scripts
```

Comment:
which ones?

Reply:
The ones in vhdbuilder/packer/generate-windows-vhd-configuration.ps1 with the prefix "https://github.com/". They are not signed.

Reply:
Remove them from cache.